mythril/analysis/modules/external_calls.py

```diff
@@ -31,11 +33,15 @@ def _analyze_state(state):
     node = state.node
     gas = state.mstate.stack[-1]
     to = state.mstate.stack[-2]
-
+    issues = []
```

Comment:
this is just a local variable rite?

Reply:
yes, issues are retuned in this function.